mixes/reducer.js

```diff
@@ -171,6 +175,56 @@ function createReducer (config) {
         Effects.constant(navigateToMix(newMix.id))
       ]))
     },
+    [reorderPrimaryTrack]: (state, action) => {
```

Comment:
this function is complex because we need to be REALLY careful about how we move around channels in our mix. specifically, when moving tracks, we want to move it WITH its trailing transition.

note this is distinct from a normal list because we cant just have an `order` property - we calculate that from `startBeat`